packages/salesforcedx-utils/src/types/localization/messageArgs.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Maps placeholders to types
+export type PlaceholderToType<S extends string> = S extends '%s'
+  ? string
+  : S extends '%d'
+    ? number
+    : S extends '%i'
+      ? number
+      : S extends '%f'
+        ? number
+        : S extends '%j'
+          ? any
+          : unknown;
+
+// Recursively extracts argument types from a message string
+export type ExtractArgs<S extends string, Acc extends any[] = []> = S extends `${infer _}%${infer P}${infer Rest}`
```

Comment:
there's this linter warning `'_' is defined but never used.`

Why is the `_` required?  What does it do?